.github/renovate.json

```diff
@@ -76,6 +76,14 @@
       ],
       "enabled": false
     },
+    {
+      "description": "Exclude react-router-dom until refactoring is done in Operate (see issue #17736)",
+      "matchManagers": ["npm", "nvm"],
+      "excludePackagePatterns": [
```

Comment:
does it need to be exclude or `matchPackagePatterns` here? Beacuse I want it **not** to be enabled :exploding_head:

Reply:
Won't this disable for Tasklist too?

Reply:
It does need to be `matchPackagePatterns` because you want this rule you are writing to match "when manager == npm || nvm" and "when package pattern == react-router-dom".

With your current rule you'd disable Renovate for everything except this one dependency 😅 

> Won't this disable for Tasklist too?

This is indeed correct.

Reply:
@cmur
> With your current rule you'd disable Renovate for everything except this one dependency 😅
It's updated

> You can use https://docs.renovatebot.com/configuration-options/#matchfilenames to scope only for the Operate package.json (or similar)
added the scope